app/views/tagging_spreadsheets/index.html.erb

```diff
@@ -15,7 +17,9 @@
     <% @tagging_spreadsheets.each do |spreadsheet| %>
       <tr>
         <td><%= spreadsheet.url %></td>
-        <td><%= spreadsheet.created_at %></td>
+        <td><%= spreadsheet.description %></td>
+        <td><%= spreadsheet.created_at.to_formatted_s(:long) %></td>
```

Comment:
I have a PR related to this line (https://github.com/alphagov/content-tagger/pull/115) what do you think of leaving it as-is here and then we can merge both PRs together?